kythe/typescript/indexer.ts

```diff
@@ -508,7 +508,7 @@ class StandardIndexerContext implements IndexerHost {
     // TODO: update symbolNames table to account for context kind
 
     if (!declarations || declarations.length < 1) {
-      throw new Error('TODO: symbol has no declarations?');
+      console.warn(`TODO: symbol ${sym.name} has no declarations`);
```

Comment:
Use the `todo` function here.

Reply:
unfortunately there is no `ts.Node` to pass to the todo function in this case because the only context is a symbol.

Reply:
Could you add a comment explaining this for future readers?

Reply:
Done.